tests/lit.cfg

```diff
@@ -57,13 +57,18 @@ config.test_exec_root = os.path.join(llbuild_obj_root, 'tests')
 #
 # FIXME: This is pretty compiler specific and probably should just be ripped out
 # of lit.
-config.target_triple = None
+config.target_triple = ""
```

Comment:
I've had to change this, as we get an error running the tests, as lit assumes that config.target_triple cannot be None:

```python
            # If this is a part of the target triple, it fails.
            if item and item in self.suite.config.target_triple:
                return True
```

https://github.com/llvm-mirror/llvm/tree/master/utils/lit/lit/Test.py#L240